pypeit/images/combineimage.py

```diff
@@ -156,6 +159,13 @@ def run(self, process_steps, bias, pixel_flat=None, illum_flat=None,
                 pypeitImage.mask[indx] = pypeitImage.bitmask.turn_off(pypeitImage.mask[indx], 'SATURATION')
             mask_stack[kk, :, :] = pypeitImage.mask
 
+        # Check that the lamps being combined are all the same:
+        if not lampstat[1:] == lampstat[:-1]:
+            msgs.warn("The following files contain different lamp status")
+            for file in self.files:
+                print(msgs.indent() + file)
+            msgs.error("Unable to combine frames with different lamp status")
```

Comment:
Can we print a list of the files to the screen so the user knows which ones to inspect?

Reply:
I might be misunderstanding (line 166 prints the filenames), so I've now updated this printout to indicate the lamp status as well (and slightly prettier printout).